settings/common.py

```diff
@@ -297,3 +298,9 @@
 REDOC_SETTINGS = {
     'SPEC_URL': ('docs.yaml', {'format': '.yaml'}),
 }
+
+#Prevents Datetime warning by showing errors 
+warnings.filterwarnings(
```

Comment:
@deshraj  `filterwarnings()` helped me to debug the code as it converted warnings into errors. I think it would be helpful for other developers if they face similar issues in future development.